pkg/chain/celo/celoutil/celoutil.go

```diff
@@ -19,10 +19,16 @@ import (
 
 var logger = log.Logger("keep-celoutil")
 
-// CeloClient wraps the core `bind.ContractBackend` interface with
+// ABI for errors bubbled out from revert calls. Not used directly as errors are
+// neither encoded strictly as method calls nor strictly as return values, nor
+// strictly as events, but some various bits of it are used for unpacking the
+// errors. See ResolveError below.
+const errorABIString = "[{\"constant\":true,\"outputs\":[{\"type\":\"string\"}],\"inputs\":[{\"name\":\"message\", \"type\":\"string\"}],\"name\":\"Error\", \"type\": \"function\"}]"
```

Comment:
Do we know why is it different between Ethereum and Celo? Is it because of `go-ethereum` and `celo-blockchain` dependency versions? If we update dependencies, can we have the same `errorABIString`?

Reply:
It's because `celo-blockchain` has been forked from an older version of `go-ethereum` than we use. Update of dependencies won't change anything.